pkg/auth/rbac_test.go

```diff
@@ -87,3 +90,74 @@ func TestValidateRbacPermission(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckUserPermissions(t *testing.T) {
+	tcs := []struct {
+		Name        string
+		rbacConfig  RBACConfig
+		user        *User
+		env         string
+		envGroup    string
+		application string
+		action      string
+		WantError   string
+	}{
+		{
+			Name:        "Check user permission works as expected",
+			user:        &User{DexAuthContext: &DexAuthContext{Role: "Developer"}},
+			env:         "production",
+			envGroup:    "production",
+			application: "app1",
+			action:      PermissionCreateLock,
+			rbacConfig: RBACConfig{DexEnabled: true, Policy: map[string]*Permission{
```

Comment:
I'm surprised the CI does not complain about the formatting here. Please check that `RBACConfig` indentation is correct.

Reply:
Yeah... Apparently having the policy on a different line causes this strange formatting. I moved the policy to the same line.